server/src/main/java/org/elasticsearch/script/IngestScript.java

```diff
@@ -30,7 +33,8 @@
     public static final String[] PARAMETERS = { "ctx" };
 
     /** The context used to compile {@link IngestScript} factories. */
-    public static final ScriptContext<Factory> CONTEXT = new ScriptContext<>("ingest", Factory.class);
+    public static final ScriptContext<Factory> CONTEXT = new ScriptContext<>("ingest", Factory.class,
+        200, TimeValue.timeValueMillis(0), new Tuple<>(375, TimeValue.timeValueMinutes(5)));
```

Comment:
While I understand the reasoning for increasing this specifically for ingest (beats modules using many ingest scripts), allowing so high numbers seems like we are easily allowing thrashing, especially since one ingest request can have dozens of scripts. Can you please open a followup issue to investigate ways to allow ingest to utilize many scripts in one request without tripping the limit?

Also, don't we need a similar change for IngestConditionalScript, which are more likely to be unique and numerous?

Reply:
> Can you please open a followup issue to investigate ways to allow ingest to utilize many scripts in one request without tripping the limit?

https://github.com/elastic/elasticsearch/issues/53896

> Also, don't we need a similar change for IngestConditionalScript, which are more likely to be unique and numerous?

[Added](https://github.com/elastic/elasticsearch/pull/53765/files#diff-8ac8802ce2b117ef9659887a33efbcdfR35)